client/new-stack-editor/lib/views/statusbar.coffee

```diff
@@ -0,0 +1,18 @@
+kd = require 'kd'
+JView = require 'app/jview'
+
+
+module.exports = class Statusbar extends JView
+
+
+  constructor: (options = {}, data) ->
+
+    options.cssClass = kd.utils.curry 'statusbar', options.cssClass
+    data ?= { row: 1, column: 1, title: '...' }
+
+    super options, data
+
+
+  pistachio: ->
+    '{h3{#(title)}} <caret>{{#(row)}}:{{#(column)}}</caret>'
```

Comment:
I agree, but then you never know how other browsers will behave. Just recently fixed IE edge problems with the `<main>` tag that's why commented. You can do sth like `'{h3{#(title)}} <span>{{#(row)}}:{{#(column)}}</span>'`